client/service/service.go

```diff
@@ -78,7 +78,7 @@ type Option func(*Service)
 // New starts a new Service with options.
 func New(options ...Option) (*Service, error) {
 	dialKeepaliveOpt := grpc.WithKeepaliveParams(keepalive.ClientParameters{
-		Time: 1 * time.Minute,
+		Time: 5 * time.Minute,
```

Comment:
If `EnforcementPolicy.MinTime` is set to 5 min maybe we should set this a little bit higher, or decrease MinTime on the server?

Reply:
@krhubert What did we say about client time again?

Reply:
also using this variable to explain the why 5 could be good

`Time: EnforcementPolicy.MinTime + Time.epsilon`

I don't know if epsilon exists in go but something like that to add a small delay

Reply:
I think `EnforcementPolicy.MinTime` is enforce for at least 3 ping in this period. So even if `ClientParameters.Time` = `EnforcementPolicy.MinTime` it's find.

Reply:
EnforcementPolicy.MinTime - 1 min
Time for server - 1 min
Time for client - 2 min

Reply:
See https://forum.mesg.com/t/reduce-keepalive-values/247